src/public-key.js

```diff
@@ -0,0 +1,40 @@
+const URL = require('url');
+const fetch = require('node-fetch');
+
+const CACHE = {};
+const BASE_URL = 'https://handyman.ambassify.eu/eventbus/';
```

Comment:
I don't think we should expose our own `BASE_URL` in the package.
Require users to set it explicitly.